doc.go

```diff
@@ -214,6 +222,18 @@ For instance:
     c.Assert("these are the voyages", qt.Matches, `these are .*`)
     c.Assert(net.ParseIP("1.2.3.4"), qt.Matches, `1.*`)
 
+NilError
```

Comment:
Please remove this and modify IsNil doc.

Reply:
done